aptoide-installer/src/main/java/cm/aptoide/pt/installer/AptoideInstaller.kt

```diff
@@ -79,7 +79,7 @@ class AptoideInstaller @Inject constructor(
           val totalObbSize = obbFiles.totalLength
           val apkFraction = 49.0 * totalApkSize / (totalApkSize + totalObbSize)
           val obbFraction = 49.0 * totalObbSize / (totalApkSize + totalObbSize)
-          if(totalObbSize > 0){
+          if (totalObbSize > 0) {
```

Comment:
Also adjust total size calculations according to the actual files that will be used.
I would suggest to filter unnecessary ones even before going into the installation process itself.
Some kind of extension function on `InstallPackageInfo` that will be used both by downloader and installer for filtering purpose.

Reply:
We decided to filter in the mapper right after mapping everything